Exiled.API/Features/Player.cs

```diff
@@ -358,11 +358,6 @@ public Camera079 Camera
             set => SetCamera(value.cameraId);
         }
 
-        /// <summary>
-        /// Gets a value indicating whether the player's role type is any NTF type <see cref="ReferenceHub"/>.
-        /// </summary>
-        public bool IsNTF => Team == Team.MTF;
```

Comment:
Don't remove API like this, as this breaks plugins. Add the `[Obsolete("Message here")]` attribute and it'll be removed before the next full release.

Reply:
should I make a new commit?